homeassistant/components/tractive/strings.json

```diff
@@ -19,15 +19,46 @@
     }
   },
   "entity": {
+    "binary_sensor": {
+      "tracker_battery_charging": {
+        "name": "Tracker battery charging"
+      }
+    },
+    "device_tracker": {
+      "tracker": {
+        "name": "Tracker"
+      }
+    },
     "sensor": {
+      "daily_goal": {
+        "name": "Daily goal"
+      },
+      "minutes_active": {
+        "name": "Minutes active"
+      },
+      "tracker_battery_level": {
+        "name": "Tracker battery"
```

Comment:
Any reason you're prefixing these names with "Tracker"? Isn't the device a tracker?

Reply:
Yes, the full name of this entity is `<pet name> Tracker battery`. If we remove the `tracker` prefix, we get `<pet name> battery` which I think is a strange wording. The pet's name is used as a prefix in the name of each entity except a `device_tracker`.